stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -152,14 +154,14 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         MessageLongClickListenerView { _, view ->
             optionsView = view
 
-            binding.blurLayer.isVisible = true
+            binding.shadowLayer.isVisible = true
 
             if (view.parent != null && view.parent is ViewGroup) {
```

Comment:
Nice idea to take messages straight from the message list. But can you explain when this view is added again to the RecyclerView?

Reply:
I'm not entirely sure about this solution. It takes the entire holder as-is, with the timestamp, avatar, replies, seen indicators included (the replies arrow is nearly invisible, grey on grey here)

![image](https://user-images.githubusercontent.com/12054216/101617260-ba096e00-3a10-11eb-83e8-f529e65eaa71.png)

The design only has the message bubble and the avatar:

![image](https://user-images.githubusercontent.com/12054216/101617528-0359bd80-3a11-11eb-8780-48f392de4610.png)

I think we should use a separate View here instead of reusing the old one. It would let us solve things like truncating overly long messages too. I see that we are going with a `ScrollView` for that, but that feels like an odd solution, I'd rather do maxlines and ellipsize in that case (the design doesn't seem to address it directly).

Reply:
List message is not added anymore, but you can see that the recyclerview doesn't lose it. Perhaps the recycler view recreate its views after this one is removed. 

About the recreation using the same view or recreating it using the ViewHolder, this review seams more related to the PR #932 then this one. This one just add the message options list in the bottom of the message in the new screen. 

To avoid using this PR to solve more than it is intended, what about fix this matter on a next PR?